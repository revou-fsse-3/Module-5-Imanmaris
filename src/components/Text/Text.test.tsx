
import {render, screen} from '@testing-library/react';
import Text from '.';

describe ('Unit Test for Text Component', () => {

    test('component should be return text', () => {
        
        render(<Text>{'testing text'}</Text>);
        const element = screen.getByText('testing text')
        expect(element).toMatchSnapshot();
      })
    
});