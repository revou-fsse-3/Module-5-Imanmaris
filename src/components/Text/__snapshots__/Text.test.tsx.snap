// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unit Test for Text Component component should be return text 1`] = `
<p>
  testing text
</p>
`;
